ui/js/auth.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const authApp = angular.module('auth', ['LocalStorageModule', 'auth0.auth0', 'react']);
+
+authApp.config(['angularAuth0Provider', 'localStorageServiceProvider', '$locationProvider',
```

Comment:
Would you mind added this as a purely React entrypoint instead? (eg like for `ui/test-view/index.js`)

Auth0 docs have some examples for React here:
https://auth0.com/docs/quickstart/spa/react

(Our use of React components embedded in an AngularJS app for the other entrypoints is purely as a transition step, rather than something we like doing hehe 😄)